CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-# dbt_quickbooks version.version
+# dbt_quickbooks v0.18.0
+This release introduces the following updates.
+
+## Bug Fixes
+- Modified `int_quickbooks__bill_payment_double_entry model` to ensure bill payment accuracy by updating the `converted_amount` logic. These changes improve the integrity and granularity of debit/credit entries for bill payments. ([PR #155](https://github.com/fivetran/dbt_quickbooks/pull/155))
+  - Utilized the line-level `amount` from bill payment lines instead of `total_amount` from bill payments, then aggregated that upward. Bill payment lines in a bill payment can have different exchange rates applied depending on which bill it's associated with, so the bill exchange rate can come into play.
```

Comment:
> then aggregated that upward

Could you clarify what this means? Does that mean something like summing the lines to calculate the total or something else?

Reply:
Yep essentially! Updated.